### PR TITLE
resolves #16 introduce a new command to generate exercises PDF

### LINKED ISSUE #16: Introduce a new command to generate an exercise PDF
Here's a summary:

Name | Intended users | Document type | File type | Generate | 
------------ | ------------- | ------------- | ------------- | -------------
Presentation | Trainers | reveal.js presentation | HTML | `./gradlew convertSlides`
Presentation support | Students | reveal.js presentation without speaker notes | PDF | manual
Online course | Students | HTML page | HTML | `./gradlew convertOnlineHtml`
course PDF | Students | Book document | PDF | `./gradlew convertOnlinePdf`
exercise PDF | Students | Article document | PDF | -

We should introduce a new command to generate exercise PDF: `./gradlew convertExercisePdf`.
The PDF will be generated in `build/exercise/*.pdf`.

Exercices should be written in separated AsciiDoc documents:

```
modules/4.0-neo4j-admin/docs/02_OverviewOfNeo4jAdministration.adoc
modules/4.0-neo4j-admin/docs/exercises/01_SettingUpNeo4jEnterpriseEditionEC2.adoc
```

The reason is that we want to convert only the exercises (not all the documents that might or might not contain an exercise).

The Gradle command will convert all the document in the `modules/4.0-neo4j-admin/docs/exercises/` folder.

### PR DESCRIPTION
**Pull request by Mogztter**
resolves #16

### CODE REVIEW

File: modules/4.0-neo4j-admin/docs/02_OverviewOfNeo4jAdministration.adoc [hunk 1]
```diff
@@ -52,7 +51,7 @@ There are also  a set of other language drivers written by the Neo4j community.
 A Neo4j client can only access a single database at a time.
 --
 
-image::{imagedir}/bolt-diagram.png[bolt-diagram,width=400,align=center]
+image::bolt-diagram.png[bolt-diagram,width=400,align=center]
```
Comment (Mogztter) [inline modules/4.0-neo4j-admin/docs/02_OverviewOfNeo4jAdministration.adoc]: We should not prefix image target with `{imagedir}` but use the built-in `{imagesdir}` attribute, see #17

File: modules/4.0-neo4j-admin/docs/exercises/SettingUpNeo4jEnterpriseEditionEC2.adoc [hunk 2]
```diff
@@ -0,0 +1,87 @@
+ifndef::imagesdir[:imagesdir: ../../images]
```
Comment (Mogztter) [inline modules/4.0-neo4j-admin/docs/exercises/SettingUpNeo4jEnterpriseEditionEC2.adoc]: I'm using the condition to define `imagesdir` when the file is preview locally (ie. using the browser extension or an IDE plugin)

**Comment by Mogztter** [comment]
@ElaineRosenberg Actually we might not need another task because the `convertOnlinePdf` also converts documents in subdirectory. So the command `./gradlew :convertOnlinePdf` will generate a PDF file at _modules/4.0-neo4j-admin/build/online/exercises/*.pdf_.
Do you still want to create a dedicated task?

**Comment by ElaineRosenberg** [comment]
If it will automatically create the PDFs in the exercise directory, then we no longer need a specific task.

**Comment by Mogztter** [comment]
> If it will automatically create the PDFs in the exercise directory, then we no longer need a specific task.

Ok.
This pull request now has the following changes:
- Exercise section is now in a dedicated file _docs/exercises/SettingUpNeo4jEnterpriseEditionEC2.adoc_
- Generates a PDF in _build/online/exercises_
- Replaces `{imagedir}` by `{imagesdir}` (built-in)
- Uses an `include` directive with `leveloffset` to include the exercise section in the main document

**Comment by Mogztter** [comment]
@ElaineRosenberg I think you've already integrated these changes on master, closing.

**Pull request closed without merging**